tests/page-click-timeout-4.spec.ts

```diff
@@ -29,3 +29,19 @@ it('should timeout waiting for stable position', async ({page, server}) => {
   expect(error.message).toContain('waiting for element to be visible, enabled and stable');
   expect(error.message).toContain('element is not stable - waiting');
 });
+
+it('should click for the second time after first timeout', async ({page, server, mode}) => {
+  it.skip(mode !== 'default');
+
+  await page.goto(server.PREFIX + '/input/button.html');
+  const __testHookBeforePointerAction = () => new Promise(f => setTimeout(f, 2000));
```

Comment:
```suggestion
  const __testHookBeforePointerAction = () => new Promise(f => setTimeout(f, 1500));
```

This is supposed to test that a timing out click does not actually click right? If it was broken, it would be racing against the page.click on line 45, because they both happen at 2 seconds.

Reply:
Great catch!